scripts/beekeeper.pl

```diff
@@ -71,6 +71,7 @@ sub main {
     my $unblock_semaphored_jobs     = 0;    # Mark SEMAPHORED jobs to READY
     my $forgive_failed_jobs         = 0;    # Mark FAILED jobs to DONE
     my $discard_ready_jobs          = 0;    # Mark READY jobs to DONE
+    my $reset_job_for_input_id      = 0;
```

Comment:
It should probably be `jobs`, not `job`. Would be good to clarify which statuses are changed and to what.